sys/hoon.hoon

```diff
@@ -6186,12 +6186,19 @@
 ++  mute                                                ::  untyped virtual
   |=  taq/_=>(~ ^?(|.(**)))
   ^-  (each * (list tank))
-  =+  ton=(mock [taq 9 2 0 1] |=({* *} ~))
+  =+  ton=(mock [taq %9 2 %0 1] |=({* *} ~))
   ?-  -.ton
     $0  [%& p.ton]
     $1  [%| (turn p.ton |=(a/* (smyt (path a))))]
     $2  [%| p.ton]
   ==
+::  +slum: slam a gate on a sample using raw nock, untyped
+::
+++  slum
+  ~/  %slum
+  |=  [gat=* sam=*]
+  ^-  *
+  .*(gat(+< sam) [%9 2 %0 1])
```

Comment:
You don't like the explicit `%10`? It compiles to the same thing, of course.

Reply:
I do like the explicit %10 better. Let me try it that way.